src/pages/_error.tsx

```diff
@@ -1,52 +1,70 @@
-                        <p className="govuk-body">If you entered a web address, please check it was correct.</p>
-                        <p className="govuk-body">Click continue to start again.</p>
-                    </div>
-                )}
+                <div className="govuk-grid-column-two-thirds">
+                    {error ? (
+                        <div>
+                            <h1 className="govuk-heading-xl">Sorry, there is a problem with the service.</h1>
+                            <p className="govuk-body">Try again later.</p>
+                            <p className="govuk-body">
+                                Your answers have not been saved, click continue to start again.
+                            </p>
+                            <p className="govuk-body">
+                                Contact us with the feedback button at the top of this page for assistance.
+                            </p>
+                        </div>
+                    ) : (
+                        <div>
+                            <h1 className="govuk-heading-xl">Page not found</h1>
+                            <p className="govuk-body">If you typed the web address, check it is correct.</p>
+                            <p className="govuk-body">
+                                If you pasted the web address, check you copied the entire address.
+                            </p>
+                            <p className="govuk-body">
+                                If the web address is correct or you selected a link or button,{' '}
+                                <a className="govuk-link" id="feedback_link" href={FEEDBACK_LINK}>
+                                    contact
+                                </a>{' '}
+                                us about your fares data.
+                            </p>
+                            <p className="govuk-body">Click continue to start again.</p>
+                        </div>
+                    )}
 
-                <br />
-                <a
-                    href="operator"
-                    role="button"
-                    draggable="false"
-                    className="govuk-button govuk-button--start"
-                    data-module="govuk-button"
-                >
-                    Continue
-                </a>
+                    <br />
+                    <a
+                        href="operator"
+                        role="button"
+                        draggable="false"
+                        className="govuk-button govuk-button--start"
+                        data-module="govuk-button"
+                    >
+                        Continue
+                    </a>
+                </div>
             </div>
         </main>
     </Layout>
 );
 
 export const getServerSideProps = (ctx: NextPageContext): {} => {
-    return { props: { statusCode: ctx?.res?.statusCode } };
+    if (ctx.res?.statusCode === 404) {
```

Comment:
Next have a way to create a custom 404 page directly now https://nextjs.org/docs/advanced-features/custom-error-page. If we do it that way we can remove this logic around status codes.